engine/src/main/java/org/terasology/engine/world/block/internal/BlockManagerImpl.java

```diff
@@ -156,7 +157,8 @@ public void receiveFamilyRegistration(BlockUri familyUri, Map<String, Integer> r
                     if (id != null) {
                         block.setId((short) id.intValue());
                     } else {
-                        logger.error("Missing id for block {} in registered family {}", block.getURI(), familyUri);
+                        logger.atError().addArgument(() -> block.getURI()).addArgument(() -> familyUri).
+                                log("Missing id for block {} in registered family {}");
```

Comment:
same reasoning as above
```suggestion
                        logger.error("Missing id for block {} in registered family {}", block.getURI(), familyUri); //NOPMD
```